CMakePresets.json

```diff
@@ -74,28 +70,28 @@
       "hidden": true,
       "cacheVariables": {
         "HDF5_ENABLE_PLUGIN_SUPPORT": "ON",
-        "PLUGIN_TGZ_ORIGPATH": {"type": "STRING", "value": "https://github.com/HDFGroup/hdf5_plugins/archive/refs/tags"},
-        "PLUGIN_TGZ_ORIGNAME": {"type": "STRING", "value": "hdf5_plugins-master.tar.gz"}
+        "H5PL_ALLOW_EXTERNAL_SUPPORT": {"type": "STRING", "value": "TGZ"},
+        "PLUGIN_PACKAGE_NAME": {"type": "STRING", "value": "pl"},
+        "PLUGIN_TGZ_ORIGPATH": {"type": "STRING", "value": "https://github.com/HDFGroup/hdf5_plugins/releases/download/snapshot"},
+        "PLUGIN_TGZ_NAME": {"type": "STRING", "value": "hdf5_plugins-master.tar.gz"}
       }
     },
     {
       "name": "ci-StdPlugins",
       "hidden": true,
       "inherits": ["ci-base-plugins", "ci-PluginsVars", "ci-base-tgz"],
       "cacheVariables": {
-          "H5PL_VERS_MAJOR": "1",
-          "H5PL_VERS_MINOR": "14",
-          "H5PL_VERS_RELEASE": "3"
+        "PLUGIN_USE_LOCALCONTENT": "OFF"
       }
     },
     {
       "name": "ci-ExamplesVars",
       "hidden": true,
       "cacheVariables": {
-        "HDF5_EXAMPLES_COMPRESSED": {"type": "STRING", "value": "hdf5-examples-2.0.4.tar.gz"},
-        "HDF5_EXAMPLES_COMPRESSED_DIR": {"type": "STRING", "value": "${sourceParentDir}/temp"},
-        "EXAMPLES_TGZ_ORIGPATH": {"type": "STRING", "value": "https://github.com/HDFGroup/hdf5-examples/archive/refs/tags/"},
-        "EXAMPLES_TGZ_ORIGNAME": {"type": "STRING", "value": "2.0.4.tar.gz"}
+        "HDF5_EXAMPLES_COMPRESSED": {"type": "STRING", "value": "hdf5-examples-master.tar.gz"},
```

Comment:
Some of these examples links seemed suspect going from `2.0.4` to `master` so I didn't know if this was correct or not for 1.14 as opposed to develop

Reply:
This will be gone when examples is moved in